pkg/apis/openebs.io/ndm/v1alpha1/blockdeviceclaim_types.go

```diff
@@ -96,6 +99,13 @@ const (
 	VolumeModeFileSystem BlockDeviceVolumeMode = "FileSystem"
 )
 
+// BlockDeviceNodeAttributes contains the attributes of the node from which the BD should
+// be selected for claiming
+type BlockDeviceNodeAttributes struct {
+	NodeName string `json:"nodeName,omitempty"`
```

Comment:
Can we define what is node & what is host?
Is there any priority?
How will a user decide what to use? against what not to use?
Can these be interchanged later?

Since this is the API/Schema; it will be great to present the code as doc

Reply:
updated the comments.